outlook/indico_outlook/calendar.py

```diff
@@ -183,6 +196,18 @@ def _update_calendar_entry(entry, settings):
         return False
     else:
         logger.info('Request to %s %s finished with status %r and body %r', method, path, res.status_code, res.text)
+        if res.ok and (entry.action == OutlookAction.add or (not existing and entry.action == OutlookAction.update)):
+            # successfully added or updated w/ no reference to existing entry
+            OutlookCalendarEntry.create(event, user, unique_id)
```

Comment:
Is it worth creating a separate method for this considering it's just called once?